### DISCUSSION TITLE
context deadline exceeded = ipify

### DISCUSSION THREAD

**Discussion by andybcom** [JanDeDobbeleer/oh-my-posh#5978] [Troubleshoot]
Hi
two Windows 11 Pc, both using same version of OMP, PWSH and all up to date.
Both using the same configuation and both have the IPIFY segment

one works and the other comes back with "context deadline exceeded" during the IPIFY lookup

[ERROR] 13:41:15.785 request.go:]:28 → Get "https://api.ipify.org?format=json": context deadline exceeded
[TRACE] 13:41:15.785 request.go:](https://api.ipify.org?format=json) - 1.0005405s
[ERROR] 13:41:15.785 ipify.go:Get:24 → context deadline exceeded

Changing the HTTP_TIMEOUT doesnt work either, just extends the context deadline 

curl -s https://api.ipify.org?format=json works on the pc

any ideas ?

{
          "properties": {
            "http_timeout": 1000
          },
          "cache": {
            "duration": "5m0s",
            "strategy": "folder"
          },
          "style": "powerline",
          "template": " {{ .IP }} ",
          "foreground": "#efefef",
          "powerline_symbol": "\ue0b0",
          "background": "#00AAFF",
          "type": "ipify",
          "background_templates": [
            ""
          ]
        }

**Comment by JanDeDobbeleer** [comment]
The service is down, Even clicking the link doesn't make it respond.

**Reply by andybcom** [reply]
I am getting a response, for ipv4 and ipv6 addresses from the service using a curl request or browser

**Reply by JanDeDobbeleer** [reply]
Now I do too, yesterday not so much.
